idaes/power_generation/unit_models/helm/condenser_ntu.py

```diff
@@ -221,32 +233,33 @@ def build(self):
         ########################################################################
         @self.Constraint(time, doc="Heat balance equation")
         def unit_heat_balance(b, t):
-            return 0 == self.side_1.heat[t] + self.side_2.heat[t]
+            return 0 == (self.side_1.heat[t] +
```

Comment:
Should this be using `hot_side` and `cold_side` instead of `side_1` and `side_2`?

Reply:
Yes, although it doesn't matter.  Will fix.